File: Model/ViewModel/ProductAttributeVM.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace targeteo.pl.Model.ViewModel
{
    public class ProductAttributeVM
    {
        public int ProductAttributeId { get; set; }

        public string ProductId { get; set; }

        public string Value { get; set; }

        public bool IsRequired { get; set; }

        [JsonIgnore]
        public ProductVM Product { get; set; }

        public int CategoryAttributeId { get; set; }
        public CategoryAttributeVM CategoryAttribute { get; set; }

        [JsonIgnore]
        public ICollection<AttributeValueListVM> ComboboxValues { get; set; }
    }
}

File: ClientApp/src/app/model/productAttributeVm.ts
import { ProductVm } from "./productVm";
import { CategoryAttributeVm } from "./categoryAttributeVm";
import { AttributeValueListVm } from "./attributeValueListVm";

export class ProductAttributeVm {

        public ProductAttributeId: number;

        public ProductOfAttributeId: number;

        public Fk_CategoryAttributes: number;

        public Value: string;

        public Product: ProductVm;

        public CategoryAttribute: CategoryAttributeVm;

        public ComboboxValues: AttributeValueListVm[];

        public ViewFilterType: string;

}